packages/server/lib/browsers/browser-cri-client.ts

```diff
@@ -573,6 +573,18 @@ export class BrowserCriClient {
     this.extraTargetClients.delete(targetId)
   }
 
+  async closeExtraTargets () {
+    for (const [targetId] of this.extraTargetClients) {
+      debug('Close extra target (id: %s)', targetId)
+
+      try {
+        await this.browserClient.send('Target.closeTarget', { targetId })
```

Comment:
do we actually need to wait for these to resolve? I assume we can move on since these are "extra" and it shouldn't impact execution.

Reply:
Yes, we definitely need to wait for them to close. All extra tabs/window need to be closed out before the next tests starts or it breaks test isolation.